s4e-web/cypress/page-objects/auth/login.po.ts

```diff
@@ -32,20 +35,39 @@ export class Login extends Core {
     Login
       .pageObject
       .getSubmitBtn()
-      .click();
+      .click({ force: true });
+
+    return Login;
+  }
+
+
+  static errorsCountShouldBe(count: number) {
+    cy.location('pathname').should('eq', '/login');
+
+    Login
+      .pageObject
+      .getErrors()
+      .should('have.length', count);
 
-    cy.location('pathname').should('eq', '/map/products');
     return Login;
   }
 
+  static errorLogin() {
+    cy.location('pathname').should('eq', '/login');
+
+      Login
+        .pageObject
+        .getError();
+  }
+
   static loginAs(user: User) {
     // force logout
     cy.clearLocalStorage('s4eStore');
 
-    cy.visit('/login');
     Login
       .fillForm(user)
       .sendForm();
+    cy.location('pathname').should('eq', '/map/products');
```

Comment:
I deleted this assertion from sendForm() and put here, because the negaitive login test(with empty form, or not-registered user) isn't work properly with this line in sendForm() function.